src/deep-security/lib/IdentityProvider.js

```diff
@@ -101,9 +102,9 @@ export class IdentityProvider {
         break;
 
       case IdentityProvider.COGNITO_USER_POOL_PROVIDER:
-        let jwtTokenInstance = identityMetadata.getIdToken();
-        token = jwtTokenInstance.getJwtToken();
-        expireTime = jwtTokenInstance.getExpiration().unix() * 1000;
+        let idTokenInstance = identityMetadata.getSignInUserSession().getIdToken();
+        token = idTokenInstance.getJwtToken();
+        expireTime = idTokenInstance.getExpiration().unix() * 1000;
```

Comment:
- do not use `unix` method coz momentjs was removed as dependency from cognito-identity-provider